### PR TITLE
Update Readme File

### PR DESCRIPTION
**Pull request by AlselwiAisha**
Update Readme File Add website image to be better.

### CODE REVIEW

**Comment by AlselwiAisha** [comment]
Thank you.
I edit, Is that right now?

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @AlselwiAisha, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

 ## Required Changes ♻️
## Give close attention to each point I raised below 👇🏾 

- [ ] This pull request has no summary/description and a proper title. Kindly use a more descriptive title and provide a detailed summary of the thing you did in the project.

  image

   E.g of a descriptive
    - Title: Hello Microverse Project
    - Summary: 👇🏾 
       ```
            Hello Microverse
            -  Add error checking linters.yml file
            -  Add a descriptive readme using markdown syntax
            -  Add HTML file
            -  Add CSS styling
      ```
- [ ] Linters are not running successfully, they should turn green/pass. Kindly fix this. Click on the details and see what issue it has and fix them. 
  
   image

- [ ] I can see that you used the former Microverse README template which is great but based on the project requirements, it is recommended that you use the [new Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) for all your projects in Microverse. Kindly use this template to create a professional README.md file. For this template attached, kindly do the following when customizing it to suit this project: 

   - All sections of this README template are mandatory except the FAQ which is optional.
   - Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
   - Add a project title and description that reflects what this project is about.
   - For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
   - Since this project does not require any demo, you can leave out the Live Demo section.
   - For the Getting Started Section, and the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        - Installation of Git
        - Installation of NPM
        - A working browser
        - Internet connection
        - A code editor like Visual Studio Code
        - Installation of Live Server plugging in Visual Studio Code
        - Any other tool you can think about
   - For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
   - For the Install section, kindly add instructions for setting up linters used in this project
   - For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
   - For future features, add at least things you will do in the future to improve this project.

Kindly implement all the changes listed above. It will make your Pull Request and README.md file look professional and it will give you an idea of how to set README files for most of the projects you will execute as a Microverse student. 👍 �

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1,25 +1,103 @@
-# Hello Microverse
+<a name="readme-top"></a>
```
Comment (GedeonTS) [inline README.md:2]: Linters are not running successfully. Kindly Fix that so the Linters can pass.
234559675-0e7db8ca-7c34-451f-8c9f-d55c0581bf9e

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @AlselwiAisha , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 

- Correct use of GitHub flow :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

- There are conflicts with the base branch. Kindly make sure to fix that

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TadesseDev) [inline README.md:2]: - [ ] Dear @AlselwiAisha, As suggested by the previous reviewer, linters are not running successfully. Kindly Fix that so the Linters can pass.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,103 @@
...
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-This project contains a header styled with css.
+  <h3><b>Microverse README Template</b></h3>
 
-# Built With
+</div>
```
Comment (TadesseDev) [inline README.md:10]: - [Optional] Please consider Updating the Microverse logo/text with yours if possible or remove it totally. The template is designed by Microverse for us all to use to professionally document our projects and has it filled with content related to each project we may be working on.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AlselwiAisha 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [x] Good job making most of the changes requested by the previous reviewer. 🙏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — COMMENTED** [review]
# Status: Required Changes :recycle:
Hi @AlselwiAisha 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

[REQUIRED] -  As requested by the previous reviewer, Let's fix the issues with the linters.

- Check linter:
  - Linter should be set up correctly for this repository.
  - Linter result should be green. AKA: Linter should report NO errors.
  - There should be no custom changes to linter config (as an attempt to cheat linter error reports).

Expected

image

- [ ] Usually, the error is due to a problem with the `linters.yml` file, kindly visit the [html-css linter config](https://github.com/microverseinc/linters-config/tree/master/html-css) and update your `linters.yml` to the [correct one](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml)

- [ ] Kindly ensure that you copy all the code in the file. and follow the provided linters installation instruction in this order. 🛠👍

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**